lib/core.js

```diff
@@ -58,13 +58,27 @@ declare class Object {
 
     static getOwnPropertyDescriptor<T>(o: $NotNullOrVoid, p: any): PropertyDescriptor<T> | void;
     static getOwnPropertyDescriptors(o: {}): PropertyDescriptorMap;
+    /* 
+      This is no-op.
+
+      Object.getOwnPropertyNames is actually defined in OCaml code.
+      
+      https://github.com/facebook/flow/blob/master/src/typing/statement.ml#L6308
+    */
     static getOwnPropertyNames(o: $NotNullOrVoid): Array<string>;
     static getOwnPropertySymbols(o: $NotNullOrVoid): Symbol[];
     static getPrototypeOf: Object$GetPrototypeOf;
     static is<T>(a: T, b: T): boolean;
     static isExtensible(o: $NotNullOrVoid): boolean;
     static isFrozen(o: $NotNullOrVoid): boolean;
     static isSealed(o: $NotNullOrVoid): boolean;
+    /* 
+      This is no-op.
+
+      Object.keys is actually defined in OCaml code.
+
+      https://github.com/facebook/flow/blob/master/src/typing/statement.ml#L6308
```

Comment:
Should use the permalink otherwise the line numbers won't match in the future: https://github.com/facebook/flow/blob/8ac01bc604a6827e6ee9a71b197bb974f8080049/src/typing/statement.ml#L6308

Reply:
You mean with commit hash?

Reply:
Yes